Core/include/Acts/Utilities/Intersection.hpp

```diff
@@ -232,26 +233,27 @@ namespace detail {
 /// path-limit and overstep-limit
 ///
 /// @tparam intersection_t Type of the intersection object
-/// @tparam logger_t The logger type, which defaults to std::false_type to
-/// prevent the generation of logging code
+/// @tparam logger_t The logger type, which defaults to std::false_type
+///         to prevent the generation of logging code
 ///
 /// @param intersection The intersection to check
-/// @param pLimit The path-limit
-/// @param oLimit The overstep-limit
-/// @param tolerance The tolerance that is applied to the path-limit criterion
+/// @param nearLimit The minimum distance for an intersection to be considered
+/// @param farLimit The maximum distance for an intersection to be considered
 /// @param logger A optionally supplied logger which prints out a lot of infos
-/// at VERBOSE level
+///               at VERBOSE level
 template <typename intersection_t, typename logger_t = std::false_type>
```

Comment:
You didn't touch this, but what's the `logger_t`? Can this go? I don't think it's needed.